templates/patterns/file_image/pattern-file-image.html.twig

```diff
@@ -4,9 +4,8 @@
  * File image component.
  */
 #}
-{% include '@ecl/ec-component-file' with {
-  'variant': 'image',
-  'src': src,
+{% include '@ecl/media-container' with {
```

Comment:
In twig template I see that this component support tracks. But in fact in documentation of ecl2 information is not available. Did you have already discussion with ecl/ecl-twig teams regarding not update documentation?
If this track element required - probably it should be added inside pattern.